microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,239 @@
+
+    * On macOS:
+
+        * Find the path where Microsoft Edge Canary is installed on your device. For example: `/Applications/Microsoft Edge Canary.app`.
+        * Open the **Terminal** app.
+        * Enter `/Applications/Microsoft Edge Canary.app --enable-features=msWebAppManifestSidePanel,msAddEdgeSidePaneBrandUA`.
+
+
+<!-- ====================================================================== -->
+## Enable sidebar support in your PWA
+
+To make your PWA ready for pinning to the sidebar in Microsoft Edge, use the `edge_side_panel` manifest member:
+
+1. Make sure your PWA has a web app manifest file that contains at least the `name`, `short_name`, `description`, and `icons` members.
+
+    To learn more, see [Use a web app manifest to integrate a Progressive Web App into the operating system](./web-app-manifests.md).
+
+1. Add the `edge_side_panel` member to your web app manifest. For example:
+
+    ```json
+    {
+      "name": "PWAmp music player",
+      "lang": "en-US",
+      "start_url": "/",
+      "short_name": "PWAmp",
+      "description": "A skinable music player app to play your favorite mp3 files",
+      "display": "standalone",
+      "icons": [
+        {
+          "src": "./favicon-256.png",
+          "sizes": "256x256"
+        }
+      ],
+      "edge_side_panel": {}
+    }
+    ```
+
+
+<!-- ====================================================================== -->
+## Adapt your app to the minimum sidebar width
+
+The sidebar in Microsoft Edge has a default minimum width of 376 pixels and can be resized by users. Therefore, your app's layout should support the 376 pixels minimum width and be responsive.
+
+While building your app, you can test if the app's layout is usable when narrow, and responsive, by using the **Device Emulation** tool in Microsoft Edge DevTools. To learn more, see [Verify that the webpage layout is usable when narrow](../../devtools-guide-chromium/accessibility/narrow.md).
+
+If your app's layout can't support the 376 pixels minimum width, you can define your preferred width by using the `preferred_width` property in your web app manifest. For example:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "standalone",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+When you define a preferred width in your app's manifest, the following happens:
+
+* When your app is opened in the sidebar, the sidebar is automatically resized to your preferred width.
+* Users can resize the sidebar to make it larger your preferred width, or smaller, up to the 376 pixels minimum width.
```

Comment:
```suggestion
* Users can resize the sidebar to make it larger than your preferred width, or smaller, up to the 376 pixels minimum width.
```

Reply:
this sentence could be smoother but i'll look at others before trying to draft alternatives